texts/formatted_aroundTheWorld.txt

```diff
@@ -0,0 +1,1747 @@
+"<Fogg>"
+	"*Fogg"
+"<and>"
+	"and" cnjcoo
+"<Passepartout>"
+	"*Passepartout"
+"<reach>"
+;	"reach" n sg
+;	"reach" vblex inf
+	"reach" vblex pres
+;	"reach" vblex imp
+"<Suez>"
+	"Suez" np top sg
+"<in time>"
+	"in time" adv
+;	"time" n sg
+;		"in" pr
+"<.>"
+	"." sent
+"<While>"
+;	"while" adv
+;	"while" cnjsub
+	"while" cnjadv
+;	"while" n sg
+"<disembarking>"
+;	"disembark" vblex pprs
+;	"disembark" vblex ger
+	"disembark" vblex subs
+"<in>"
+;	"in" adv
+	"in" pr
+"<Egypt>"
+	"Egypt" np top sg
+"<,>"
+	"," cm
+"<they>"
+	"prpers" prn subj p3 mf pl
+"<are>"
+	"be" vbser pres
+;	"be" vbser pres p1 pl
+;	"be" vbser pres p2 sg
+;	"be" vbser pres p2 pl
+;	"be" vbser pres p3 pl
+"<watched>"
+;	"watch" vblex pp
+	"watch" vblex past
+"<by>"
+	"by" adv
+;	"by" pr
+"<a>"
+	"a" det ind sg
+"<Scotland>"
+	"Scotland" np top sg
+"<Yard>"
+	"yard" n sg
+"<detective>"
+	"detective" n sg
+"<,>"
+	"," cm
+"<Detective>"
+	"detective" n sg
+"<Fix>"
+	"fix" n sg
+;	"fix" vblex inf
+;	"fix" vblex pres
+;	"fix" vblex imp
+"<,>"
+	"," cm
+"<who>"
+;	"who" prn itg mf sp
+	"who" prn rel an mf sp
+"<has>"
+;	"have" vbhaver pres p3 sg
+	"have" vblex pres p3 sg
+"<been>"
+	"be" vbser pp
+"<dispatched>"
+;	"dispatch" vblex pp
+	"dispatch" vblex past
+"<from>"
+	"from" pr
+"<London>"
+;	"London" np top sg
+	"London" np cog sg
+"<in>"
+	"in" adv
+;	"in" pr
+"<search>"
+;	"search" n sg
```

Comment:
This should be a noun instead of a verb. You can consider the phrase "search of a bank".